faces/3.0/_index.md

```diff
@@ -9,12 +9,15 @@ support for internationalization and accessibility.
 
 * [Jakarta Server Faces 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/3.0.0)
```

Comment:
```suggestion
* [Jakarta Server Faces 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/3.0.0/plan)
```
As per checklist, this link should be in the form of `https://projects.eclipse.org/projects/ee4j.{spec}/releases/x.y/plan`